test/check-application

```diff
@@ -61,17 +61,26 @@ def run_e2e(verbose, image, browser, cpus, memory, sit):
                                   networking=network.host(),
                                   memory_mb=memory, cpus=cpus)
     selenium = windows = None
+    networking_selenium = network.host(forward={"5901": 5901, "5902": 5902})
+    # parse libvirt forward setting string:
+    # "hostfwd=tcp:127.0.0.2:5903-:5901,hostfwd=tcp:127.0.0.2:5904-:5902"
+    # the parsing result will be a dict like:
+    # {'22': '2202', '9090': '9092', '5901': '5903', '5902': '5904'}
+    fwd_port = {}
+    for rule in networking_selenium["forwards"].split(","):
+        port_info = rule.split(":")
+        fwd_port[port_info[-1]] = port_info[-2][:-1]
```

Comment:
Please use `networking_selenium["forward"]`, then you don't need to parse the QEMU option string.